sys-power/throttled/throttled-0.10.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+inherit python-single-r1 systemd
+
+DESCRIPTION="Workaround for Intel throttling issues in Linux."
+HOMEPAGE="https://github.com/erpalma/throttled"
+SRC_URI="https://github.com/erpalma/throttled/archive/refs/tags/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
I think ~x86 should also work? but I'm fine if leave it alone (if not tested, and probably not many people using)

Reply:
I prefer not to add it blindly unless someone tested it in a chroot at least, although even better on real x86 given this is hw sensitive

Reply:
I'm pretty sure all the laptop models that this package is applicable for are amd64 anyway, see the list in https://github.com/erpalma/throttled